app/models/device.rb

```diff
@@ -34,14 +34,15 @@ def unsubscribe!
     self.save!
   end
 
-  def self.touch_device(auth_key, token)
+  def self.touch_device(auth_key, token, label)
```

Comment:
Don't change the API by focing the label to be set here. Label can either be an option (`:label => 'Something'`) or have a default `nil`. I'd prefer the option.